lib/features/thread/presentation/thread_controller.dart

```diff
@@ -838,15 +866,17 @@ class ThreadController extends BaseController {
 
   void _searchEmail() {
     emailListSearch.clear();
-
     if (_accountId != null && searchQuery != null) {
       consumeState(_searchEmailInteractor.execute(
         _accountId!,
         limit: ThreadConstants.defaultLimit,
         sort: _sortOrder,
-        filter: EmailFilterCondition(text: searchQuery!.value),
+        filter: EmailFilterCondition(
+          text: searchQuery!.value, 
+        inMailbox: _currentMailboxId),
```

Comment:
srr, my mistake, we should filter `inMailbox` in Advanced Search, please remove it

Reply:
removed